src/Clock.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable */
+import { Component } from 'react';
+
+type Props = {
+  name: string,
+};
+
+export class Clock extends Component<Props> {
+  state = {
+    date: new Date().toLocaleTimeString(),
+  };
+
+  currentTime = () => {
+    this.setState(({ date: new Date().toLocaleTimeString() }));
+  };
+
+  updateTime = () => {
+    console.info(new Date().toLocaleTimeString());
+  };
+
+  test: any = 0;
```

Comment:
Do not use 'any' as a type. Instead, provide a specific type for 'test', for example, type as 'number'.

Reply:
type number is not assignable to type 'number'.
<img width="1280" alt="Screenshot 2023-05-12 at 15 26 51" src="https://github.com/mate-academy/react_clock/assets/20277989/208f372a-48ea-4025-89a2-7588e8888892">